src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,52 @@
+expected_output = {
+    "b80-22-cap10": {
```

Comment:
please have the key for values `b80-22-cap10` and others like:
```
    'ap_name': {
        "b80-22-cap10": {
            "controller_ip": "10.10.7.177",
            (snip)

Reply:
updated!